src/realm/object-store/c_api/realm.cpp

```diff
@@ -3,6 +3,30 @@
 
 namespace realm::c_api {
 
+RLM_API bool realm_get_version_id(const realm_t* realm, bool* out_found, realm_version_id_t* out_version)
+{
+    return wrap_err([&]() {
+        util::Optional<VersionID> version = (*realm)->current_transaction_version();
+        if (version) {
+            *out_version = to_capi(version.value());
```

Comment:
Check NULL (and I would say independently between the two out parameters).

Reply:
Not 100% sure what you mean with checking for NULL here?